src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+/** Abstraction of a list that can store Tasks */
+public class TaskList {
+
+    /** List to store tasks */
+    private List<Task> list;
```

Comment:
Perhaps you could add task in the variable name to make it more identifiable like taskList or tasks?